creator.pro

```diff
@@ -23,7 +23,7 @@
 #
 #-------------------------------------------------
 
-QT += core gui network widgets
+QT += core gui network widgets core5compat

```

Comment:
I'd suggest to include this module only when Qt 6 is detected: this way you'd still allow building against Qt 5.

```suggestion
QT += core gui network widgets
greaterThan(QT_MAJOR_VERSION, 5) {
    QT += core5compat
}
```

If you decide to include this change, please compile with Qt 5 to verify that it's still ok.

Reply:
The other changes won’t work in QT5. I added this so QRegExp would work as I didn’t figure out the proper QT6 way of doing it.

Reply:
`QRegularExpression` is available in Qt 5, you should use it everywhere (and I already see changes related to that)

Reply:
As in a previous reply if I figure out how to write the regular expressions in QT6 only then this can removed completely.

Reply:
No need for this anymore. Using QT6 functions only.